### ISSUE TITLE
Compilation error during install

### ISSUE THREAD

**Issue by wouterbeek** [IBCNServices/pyRDF2Vec#4]
When I try to install pyRDF2Vec on Ubuntu I get compilation errors (see below).  This is when I run installation with `pip install pyRDF2Vec`.

```
    numpy/random/mtrand/mtrand.c:45282:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘exc_traceback’; did you mean ‘curexc_traceback’?
    45282 |     tstate->exc_traceback = local_tb;
          |             ^~~~~~~~~~~~~
          |             curexc_traceback
    error: Command "x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -Inumpy/core/include -Ibuild/src.linux-x86_64-3.7/numpy/core/include/numpy -Inumpy/core/src/private -Inumpy/core/src -Inumpy/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inumpy/core/src/umath -Inumpy/core/src/npysort -I/usr/include/python3.7m -Ibuild/src.linux-x86_64-3.7/numpy/core/src/private -Ibuild/src.linux-x86_64-3.7/numpy/core/src/npymath -Ibuild/src.linux-x86_64-3.7/numpy/core/src/private -Ibuild/src.linux-x86_64-3.7/numpy/core/src/npymath -Ibuild/src.linux-x86_64-3.7/numpy/core/src/private -Ibuild/src.linux-x86_64-3.7/numpy/core/src/npymath -c numpy/random/mtrand/mtrand.c -o build/temp.linux-x86_64-3.7/numpy/random/mtrand/mtrand.o -MMD -MF build/temp.linux-x86_64-3.7/numpy/random/mtrand/mtrand.o.d" failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-noc198ux/numpy/setup.py'"'"'; __file__='"'"'/tmp/pip-install-noc198ux/numpy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-fkhmshxf/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /home/wouter/.local/include/python3.7m/numpy Check the logs for full command output.
```

I also include the full output for completeness: [out.log](https://github.com/IBCNServices/pyRDF2Vec/files/4374619/out.log)

**Comment by GillesVandewiele** [comment]
This is a rather weird one. I did not have this on my clean Ubuntu node just recently. Could you confirm if it is fixed in version 0.0.5?

**Comment by wouterbeek** [comment]
@GillesVandewiele Thanks, I can confirm that installation works with the latest commit from master.  Closing...
